sdf/1.7/frame.sdf

```diff
@@ -1,16 +1,31 @@
 <!-- Frame -->
 <element name="frame" required="*">
-  <description>A frame of reference to which a pose is relative.</description>
+  <description>A frame of reference in which poses may be expressed.</description>
 
   <attribute name="name" type="string" default="" required="1">
-    <description>Name of the frame. This name must not match another frame defined inside the parent that this frame is attached to.</description>
+    <description>
+      Name of the frame. It must be unique whithin its scope (model/world),
+      i.e., it must not match the name of another frame or model within the same
+      scope.
+    </description>
   </attribute>
 
   <attribute name="attached_to" type="string" default="" required="*">
     <description>
-      Name of the link or frame to which this frame is attached.
-      If a frame is specified, recursively following the attached_to attributes
-      of the specified frames must lead to the name of a link, a model, or the world frame.
+      If specified, this frame is attached to the specified frame. The specified
+      frame must be within the same scope and may be defined implicitly, i.e.,
+      the name of any //frame, //model, or //link within the same scope may be
+      used.
+
+      If missing, this frame is attached to the containing scope's frame. Within
+      a //world scope this is the implicit world frame, and within a //model
+      scope this is the implicit frame defined by the model's canonical link.
+
+      A frame moves jointly with the frame it is @attached_to. This is different
+      from //pose/@relative_to. @attached_to defines how the frame is attached
+      to the scope's frame, while //pose/@relative_to defines how the frame's
```

Comment:
@scpeters My thinking here was that any frame-bearing element (//link, //joint, //model, ...) has itself a frame that is (implicitly) attached to the scope's implicit frame via it's //pose. This way, everything traces back to the scope's implicit frame.

That said, I can still change to your suggestion, if you think it is more clear.

Reply:
the initial configuration of each frame is defined by its //pose relative to the scope's implicit frame, but as time passes the frames can move relative to each other and the scope's implicit frame depending on the kinematic relationships encoded in the //joint tags. Any frames that are `@attached_to` each other have unchanging relative position and orientation. Tracing the `@attached_to` tags can lead to any link or model or world frame (not joint frames since they are automatically attached to their child link frame)

Reply:
@scpeters Ah, I see where I was wrong this should indeed be clarified; thanks for the explanation. I'll see if I can add a sentence that would have resolved my confusion from the get-go.